tests/page/page-aria-snapshot-ai.spec.ts

```diff
@@ -229,3 +229,14 @@ it('should gracefully fallback when child frame cant be captured', async ({ page
       - iframe [ref=e3]
   `);
 });
+
+it('should auto-wait for navigation', async ({ page, server }) => {
+  await page.goto(server.PREFIX + '/frames/frame.html');
+  const [, snapshot] = await Promise.all([
+    page.evaluate(() => window.location.reload()),
+    snapshotForAI(page)
+  ]);
+  expect(snapshot).toContainYaml(`
+    - generic [ref=e2]: Hi, I'm frame
```

Comment:
We should wait for the frame to load for this not to be flaky.

Reply:
Let's not use `/frame/frame.html` in the tests that don't test iframe behavior. That was confusing!